src/cards/person-card/utils.ts

```diff
@@ -1,12 +1,16 @@
 import { HassEntity } from "home-assistant-js-websocket";
 
-export function getStateIcon(entity: HassEntity) {
+export function getStateIcon(entity: HassEntity, zones:HassEntity[]) {
     const state = entity.state;
     if (state === "unknown") {
         return "mdi:help";
     } else if (state === "home") {
         return "mdi:home";
     }
+    const [zone] = zones.filter(z => state === z.attributes.friendly_name);
```

Comment:
```suggestion
    const zone = zones.find(z => state === z.attributes.friendly_name);
```
Better to use find 🙂